assets/css/style.scss

```diff
@@ -611,7 +613,7 @@ opacity: 0.7;
 
 
 .esmrmb-login {
-    background-color: #009080;
+    background-color: #5DC7D7;
```

Comment:
Maybe we can keep the original color for consistency with the ESMRMB website? Just a suggestion, you decide.